metadefender-plugin-server/src/main/scala/com/opswat/teamcity/ArtifactScanner.scala

```diff
@@ -146,6 +155,32 @@ class ArtifactScanner(config: MConfigManager) extends BuildServerAdapter {
                 post.setHeader("apikey", config.mAPIKey.mkString)
               }
 
+              val buildParams = runningBuild.getParametersProvider
+              val sandboxEnabled = (buildParams.get("system.metadefender_scan_artifact_with_sandbox") == "1" ||
+                config.mSandboxEnabled.mkString == "checked")
+
+              if (sandboxEnabled) {

```

Comment:
We should probably also add a check to see if the configured MetaDefender URL points to https://api.metadefender.com because if the user enables sandbox analysis with a custom Core instance configured the "rule: multiscan" header can make all their scan fail. (We could even add a note to the plugin that the sandbox only works with MetaDefender Cloud as for now)

Reply:
Yes, that's a good idea!

Reply:
URL check and note about MDCloud only functionality added.